contrib/swtpm/template.py

```diff
@@ -0,0 +1,61 @@
+pkgname = "swtpm"
+pkgver = "0.8.1"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = ["--with-gnutls"]
+configure_env = {"NOCONFIGURE": "1"}
```

Comment:
don't think you need this since you're not running autogen.sh and autoreconf doesn't conf twice